### ISSUE TITLE
Non-responsive and inaccurate payday UX

### ISSUE THREAD

**Issue by droidmonkey** [liberapay/liberapay.com#1392]
The receiving tab where you can setup the payday information is completely baffling to me. Perhaps there is an error in our team account in the database? The following screenshot clearly shows that our team's weekly take is $8.48. I set my share to $10 and my teammate set it to 6E. No matter what I set my nominal take to, the actual take always shows $0.58. There is a leftover of $7.91. What the heck is going on???

![image](https://user-images.githubusercontent.com/2809491/51446050-64bbfa80-1cda-11e9-94c7-77276b05c17b.png)

**Comment by Changaco** [comment]
I think the algorithm that computes the "actual takes" is producing incorrect results because most donations to the KeePassXC team are in EUR but your personal account is configured to accept only USD: https://liberapay.com/droidmonkey/edit/currencies

**Comment by droidmonkey** [comment]
Yes that was the issue, I was not "accepting" Euros so it didn't add to my actual take. The UX is still confusing regarding the receiving tab. It should show how much from each currency you are receiving (or attempting to receive) and perform the "preferred currency" conversion to normalize everything for easy comparison.
